pycm/pycm_class_func.py

```diff
@@ -5,6 +5,27 @@
 from .pycm_interpret import *
 
 
+def NB_calc(TP, FP, POP, w):
```

Comment:
Shouldn't we set a default value like 1 for w?
`w=1`

Reply:
Good point 💯 
I added a default value for `w` in `NB` method https://github.com/sepandhaghighi/pycm/pull/265/commits/14dd45a2c7b93e96d516e849885c592d69678316